service/controller/clusterapi/v29/resource/clusterazs/create.go

```diff
@@ -161,9 +165,6 @@ func (r *Resource) ensureAZsAreAssignedWithSubnet(ctx context.Context, tccpSubne
 
 			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("availability zone %q already has subnet allocated: %q", az, parentNet.String()))
 
-			// TODO(marcel): Append AZ here to separate map to gather existing
```

Comment:
I wrote an extra function for this, it was very messy otherwise. Will refactor this in the main PR when Tims changes are in.